kfs-purap/src/main/java/edu/arizona/kfs/module/purap/pdf/PurchaseOrderPdf.java

```diff
@@ -0,0 +1,918 @@
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * 
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package edu.arizona.kfs.module.purap.pdf;
+
+import java.io.ByteArrayOutputStream;
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.text.NumberFormat;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Locale;
+
+import org.apache.commons.lang.ArrayUtils;
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.kuali.kfs.module.purap.PurapConstants;
+import org.kuali.kfs.module.purap.businessobject.PurchaseOrderItem;
+import org.kuali.kfs.module.purap.businessobject.PurchaseOrderVendorStipulation;
+import org.kuali.kfs.module.purap.document.PurchaseOrderDocument;
+import org.kuali.kfs.module.purap.document.PurchaseOrderRetransmitDocument;
+import org.kuali.kfs.module.purap.exception.PurError;
+import org.kuali.kfs.module.purap.pdf.PurchaseOrderParameters;
+import org.kuali.kfs.module.purap.pdf.PurchaseOrderTransmitParameters;
+import org.kuali.kfs.module.purap.util.PurApDateFormatUtils;
+import org.kuali.kfs.sys.KFSConstants;
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+import com.lowagie.text.Chunk;
+import com.lowagie.text.Document;
+import com.lowagie.text.DocumentException;
+import com.lowagie.text.Element;
+import com.lowagie.text.Font;
+import com.lowagie.text.FontFactory;
+import com.lowagie.text.Image;
+import com.lowagie.text.Paragraph;
+import com.lowagie.text.pdf.PdfPCell;
+import com.lowagie.text.pdf.PdfPTable;
+import com.lowagie.text.pdf.PdfWriter;
+
+/**
+ * Base class to handle pdf for purchase order documents.
+ */
+public class PurchaseOrderPdf extends org.kuali.kfs.module.purap.pdf.PurchaseOrderPdf {
+    private static Log LOG = LogFactory.getLog(PurchaseOrderPdf.class);
+    
+    Font ver_4_normal = FontFactory.getFont("VERDANA", 4, 0);
+    Font ver_5_normal = FontFactory.getFont("VERDANA", 5, 0);
+    Font ver_6_normal = FontFactory.getFont("VERDANA", 6, 0);
+    Font ver_10_normal = FontFactory.getFont("VERDANA", 10, 0);
+    Font ver_11_normal = FontFactory.getFont("VERDANA", 11, 0);
+    Font ver_15_normal = FontFactory.getFont("VERDANA", 15, 0);
+    Font cour_7_normal = FontFactory.getFont("COURIER", 7, 0);
+    Font cour_16_bold = FontFactory.getFont("COURIER", 16, 1);
+    static KualiDecimal zero = KualiDecimal.ZERO;
```

Comment:
Variables ver_11_normal, ver_15_normal and cour_16_bold are not used. Please remove.